pom.xml

```diff
@@ -941,6 +953,9 @@ limitations under the License.
             <additionalOptions>
               <additionalOption>${maven-javadoc-html5}</additionalOption>
             </additionalOptions>
+            <additionalJOptions>
+              <additionalJOption>-J-Xss4m</additionalJOption>
```

Comment:
Is this an intentional change? Why is it required?

Reply:
Increasing amount of keywords could lead to StackOverflowError during Travis CI, both at compile time and javadoc generation time, This is a workaround for javadoc, however I am not sure if the issue still exists after commit https://github.com/apache/calcite/commit/561c6427e3602510d619b45cd7da840773fa0d2c, I will make a test.